regression/tests/lms/test_instructor_dashboard.py

```diff
@@ -42,17 +40,3 @@ def setUp(self):
         self.dashboard_page.select_course(get_course_display_name())
         self.course_page.wait_for_page()
         self.instructor_dashboard.visit()
```

Comment:
REVIEWERS: It is unclear whether or not it is worthwhile to have an Instructor Dashboard visit test, which this setup included, so I have left this for now.